ui/lib/core/addon/templates/components/readonly-form-field.hbs

```diff
@@ -0,0 +1,57 @@
+{{#unless
+  (or
+    (eq attr.type "boolean")
+    (contains
+      attr.options.editType
+      (array
+        "boolean"
+        "optionalText"
+        "searchSelect"
+        "mountAccessor"
+        "kv"
+        "file"
+        "ttl"
+        "stringArray"
+        "json"
+      )
+    )
+  )
+}}
+  <label for="{{attr.name}}" class="is-label" data-test-readonly-label>
+    {{labelString}}
+    {{#if attr.options.helpText}}
+      {{#info-tooltip}}
+        <span data-test-help-text>
+          {{attr.options.helpText}}
+        </span>
+      {{/info-tooltip}}
+    {{/if}}
+  </label>
+  {{log attr "what?"}}
```

Comment:
maybe a leftover from debugging?

Reply:
Haha yes, struggling with the readonly form field test. I thought I removed this before push though! 😂